tools/sigh.ts

```diff
@@ -63,8 +63,12 @@ import * as AstNode from '../../runtime/manifest-ast-nodes.js';
   }]
 };
 
+const build = buildPath('.');
+const webpack = webpackPkg('webpack');
+const webpackTools = webpackPkg('webpack-tools');
+
 const steps: {[index: string]: ((args?: string[]) => boolean)[]} = {
-  languageServer: [peg, build, webpackTools, languageServer],
+  languageServer: [peg, build, buildPath('./src/tools/aml-language-server', ['vscode-jsonrpc', 'vscode-languageserver']), webpackPkg('webpack-languageserver'), languageServer],
```

Comment:
I would extract `buildPath('./src/tools/aml-language-server', ['vscode-jsonrpc', 'vscode-languageserver'])` to a const as you did for `buildPath('.')`, will be easier to read IMO